assets/css/particle-theme.css

```diff
@@ -1097,7 +1097,7 @@ nav a em {
 
 p {
 	font-size: 17px;
-	color: #fff;
+	color: rgb(100 125 137);

```

Comment:
this changes for all p tags. did you verify other styles on the page? any other part is not afected?